modules/shared/src/main/scala/org/tessellation/currency/schema/feeTransaction.scala

```diff
@@ -21,7 +21,8 @@ object feeTransaction {
     destination: Address,
     amount: Amount,
     parent: FeeTransactionReference,
-    salt: TransactionSalt
+    salt: TransactionSalt,
+    dataUpdateHash: Hash
```

Comment:
I think that `dataUpdate` would be descriptive enough. You don't need to put `hash` in the name, the same way you don't add `sourceAddress` or `destinationAddress`

Reply:
👍🏼